### PR TITLE
add Laravel 11 support

### PR DESCRIPTION
**Pull request by atmonshi**

### CODE REVIEW

**Comment by atmonshi** [comment]
need to do more testing since sanctum updated to v4 and not sure if it will affect the pacakge! 
https://laravel.com/docs/master/upgrade#sanctum

**Comment by faizananwerali** [comment]
Laravel 11 has been released. Now waiting for this pull request to get approved.

**Comment by faizananwerali** [comment]
I have tested and it is working fine with Laravel 11 and Sanctum.

**Comment by atmonshi** [comment]
> I have tested and it is working fine with Laravel 11 and Sanctum.

prefect, no extra changes required?
I marked this PR ready :)

**Comment by faizananwerali** [comment]
> > I have tested and it is working fine with Laravel 11 and Sanctum.
> 
> prefect, no extra changes required? I marked this PR ready :)

Yes no extra changes required

**Comment by rupadana** [comment]
Thank you for your contribution @atmonshi @faizananwerali, 

We can merge it after the test pase is successfully passed. There are [error](https://github.com/rupadana/filament-api-service/actions/runs/8316144229/job/22763165590?pr=21) while running test, can you please update it?

**Comment by atmonshi** [comment]
all good now, hope I didnt forget anything
if you can run the tests and let see :)

**Comment by rupadana** [comment]
> all good now, hope I didnt forget anything if you can run the tests and let see :)

hmmm its like the migration isn't executed

**Comment by atmonshi** [comment]
I dont know what happened there , but I hope this time it will work :)
btw you have access to edit anything in this pr :)

File: tests/Fixtures/Database/Migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php
```
Comment (rupadana) [inline tests/Fixtures/Database/Migrations/2014_10_12_000000_create_users_table.php:1]: Please remove this migration

**Pull request merged**